Formula/o/openrdap.rb

```diff
@@ -0,0 +1,29 @@
+class Openrdap < Formula
```

Comment:
why not `rdap`?

Reply:
When I name it, my reasoning was that **rdap** is a protocol an this formula is for the open source query implementation of it. But can be changed.

Reply:
but the artifact name is not openradp, right? Context wise, the class name is just really helping the end users to simplify the memorization of the formula name to ease the installation process.

Reply:
i believe that you are absolutely wright! i've made the changes and submitted here.

Reply:
I'm not particularly invested in the naming here, but it would seem that the implementation is still called OpenRDAP (https://www.openrdap.org/):

> OpenRDAP is a command line client for the [Registration Data Access Protocol](https://en.wikipedia.org/wiki/Registration_Data_Access_Protocol), written in [Go](https://www.golang.org/).

I think it makes more sense for the formula name to match the project/implementation name, even if the actual binary/client program that a user would run is named differently.

On this precedent, we ship `openssh` even though the commands that it provides are `ssh`, `ssh-*`, etc.; we also ship `openldap` even though the commands that it provides are `ldap*` and `libldap.{dylib,so}`.

Perhaps I am [bikeshedding](https://thedecisionlab.com/biases/bikeshedding) here more than anything else, but I do think it's more effective to agree upon naming at formula creation time rather than through a rename later.

Reply:
yeah, I do think it is good to have a good naming upfront (that is why I use the naming convention for easy use as an agurment), using openssh and openldap as comparison is a bit unfair.

The reason why the name is all over the place, that just shows the binary is very familiar to many people.

Reply:
also just go down the openssh, openldap examples, they also use the `https://github.com/openssh/openssh-portable` and `https://github.com/openldap/openldap` for their core projects. 

@mperon if you can rename the repo from openrdap/rdap into openrdap/openrdap, I might feel like openrdap is a better name.

Reply:
We recently have a case for it as well, https://github.com/Homebrew/homebrew-core/pull/152353

This is another example about the repo name thing, https://github.com/Homebrew/homebrew-core/pull/152763

Reply:
raised an issue to ask upstream to see if they can rename the repo to avoid some confusion. :)